samples/Mvc.FileUpload/Controllers/FileController.cs

```diff
@@ -47,7 +43,9 @@ public IActionResult FormToUpload()
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> Upload()
         {
-            if (!MultipartRequestHelper.IsMultipartContentType(Request.ContentType))
+            var requestMediaType = MediaTypeHeaderValue.Parse(Request.ContentType);
```

Comment:
is requestMediaType used anywhere else in this method? If not, why not have the extension directly on HttpResponse?